src/ledger/LedgerEntryTests.cpp

```diff
@@ -1,261 +0,0 @@
-// Copyright 2015 Stellar Development Foundation and contributors. Licensed
-// under the Apache License, Version 2.0. See the COPYING file at the root
-// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
-
-#include "AccountFrame.h"
-#include "LedgerDelta.h"
-#include "OfferFrame.h"
-#include "TrustFrame.h"
-#include "crypto/SecretKey.h"
-#include "database/Database.h"
-#include "ledger/LedgerManager.h"
-#include "ledger/LedgerTestUtils.h"
-#include "lib/catch.hpp"
-#include "main/Application.h"
-#include "test/TestUtils.h"
-#include "test/test.h"
-#include "util/Logging.h"
-#include "util/Timer.h"
-#include "xdrpp/autocheck.h"
-#include "xdrpp/marshal.h"
-#include <memory>
-#include <unordered_map>
-#include <utility>
-
-using namespace stellar;
-
-namespace LedgerEntryTests
-{
-
-TEST_CASE("Ledger Entry tests", "[ledgerentry]")
-{
-    Config cfg(getTestConfig(0));
-
-    VirtualClock clock;
-    Application::pointer app = createTestApplication(clock, cfg);
-    app->start();
-    Database& db = app->getDatabase();
-
-    SECTION("round trip with database")
```

Comment:
I could not find equivalent tests (that would now be LedgerStateRoot tests that commit to the database)